src/vivarium_gates_mncnh/model_specifications/model_spec.yaml

```diff
@@ -141,7 +144,14 @@ configuration:
             include:
                 - 'age_group'
                 - 'delivery_facility_type'
-
+        postpartum_depression:
+            include:
+                - 'age_group'
```

Comment:
I don't see in the docs what stratifications are required - how did you know to stratify by these? Is it just all related variables?

Reply:
Yes it isn't specifically in the docs but we are doing it the same as the other maternal burden observers.